actions/class.Main.php

```diff
@@ -401,7 +401,22 @@ public function index()
     private function getNavigationElementsByGroup($groupId)
     {
         $entries = [];
+
+        /**
+         * @var int $i
+         * @var  Perspective $perspective
+         */
         foreach (MenuService::getPerspectivesByGroup($groupId) as $i => $perspective) {
+            if (!$this->getSectionVisibilityFilter()->isVisible($perspective->getId())) {
```

Comment:
$perspective could be a null so will result in fatal if method is called on null
```suggestion
            if ($perspective && !$this->getSectionVisibilityFilter()->isVisible($perspective->getId())) {
```